frc_characterization/logger_analyzer/data_analyzer.py

```diff
@@ -1723,7 +1737,7 @@ def _plotVoltageDomain(subset, qu, step, STATE):
     ax.set_ylabel("Acceleration")
     ax.set_title("Dynamic acceleration vs acceleration-portion voltage")
 
-    if STATE.test.get() == "Elevator":
+    if STATE.test.get() == Tests.ELEVATOR.value:
```

Comment:
Surely this function can be passed the STATE.test.get(), not just STATE?

as before, convert to the enum members then compare

Reply:
The constants ks, kv, ka, and r_square in the function are from the STATE variable.